README.md

```diff
@@ -99,7 +98,7 @@ client secret and access token.
 > :warning: Running these commands will warn you that a client secret must be
 > added for your OAuth2 provider. We'll take care of this in the next step!
 
-#### Save your Client Secret
+#### Save the Client Secret
```

Comment:
Noticed that there's a straggling `(dev)` on L110... Could we include the change to `(local)` in this pull?